scrips/eventTile.js

```diff
@@ -0,0 +1,83 @@
+import { createDomElement, appendChildren, displayTime } from "./utils.js";
+import { cellHeight, currentEventTileId } from "./calendarVars.js";
+import { openModal } from "./modal.js";
+import { constructNewEvent, getEventLength } from "./eventsData.js";
+import { currentEventDataKey, storeDataInLocalStorage } from "./handleLocalStorage.js";
+import { minutesToHour } from "./timeCalculations.js";
+
+const calculateTopPosition = (eventData) => {
```

Comment:
If I were you I would avoid adding `data` suffix here. What you have here is event entity. Even more I would not pass whole event object just the piece which is needed here which is `eventStartTime`
```suggestion
const calculateTopPosition = (event) => {
```